### PR TITLE
Game build

### PR DESCRIPTION
**Pull request by Salvador-ON**
I created two factoy functions one for the players and another one for the board

I created a Main class called Tic Tac Toe wich has the methods with the logic of the game

### CODE REVIEW

File: assets/js/app.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
+const player = (name, marker) => ({ name, marker });
+
+const gameboard = () => ({
+  arrayboard: ['', '', '',
+    '', '', '',
+    '', '', ''],
+});
+
+class TicTacToeGame {
```
Comment (mcrd25) [inline assets/js/app.js:9]: I've realized that you have DOM manipulation and aPplication logic tasks in the same class as an file.
Please remember that you ought to separate the two.
See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) from pathwright.

File: assets/js/app.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    message.id = 'message';
+    message.classList.add('text-center', 'text-warning', 'font-weight-bold', 'h2', 'pt-4');
+    message.innerHTML = `${this.playerX.name} and ${this.playerO.name} it's a tie, play agian`;
+    document.getElementById('turnDisplay').innerHTML = '';
+    document.getElementById('cont').appendChild(message);
+  }
+}
+
+
+function run() {
```
Comment (mcrd25) [inline assets/js/app.js:114]: [OPTIONAL]
I noticed that you used ES6 syntax for most of this app. Maybe it's best for uniformity for you to use ES6 here to. What do you think? 🤔 

This is optional.

Comment (Salvador-ON) [reply assets/js/app.js:114]: ok thanks i'll improve it

File: assets/js/app.js (hunk 1 above)
Comment (Salvador-ON) [reply assets/js/app.js]: thanks i will split and separate the dom and the logic

**Comment by Salvador-ON** [comment]
i split my classes to logic and dom. I also use webpack  to improt export the classes from the different classes

**Comment by Salvador-ON** [comment]
Thank you. I'll proceed to merge the pull request

**Pull request merged**